scripts/apps/authoring-react/subcomponents/content-profile-dropdown.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import {IArticle, IContentProfile} from 'superdesk-api';
+import {gettext} from 'core/utils';
+import {Spacer} from 'superdesk-ui-framework/react';
+import {sdApi} from 'api';
+
+interface IProps<T> {
+    article: T;
+    reinitialize: (itemWithChanges: T) => void;
+}
+
+interface IState {
+    profilesList: Array<IContentProfile>;
```

Comment:
if a variable is typed - do not add type related info to the variable name. `profiles` is fine here, it'd be typed as an array so we'd know what it is